Prefabs/Tanks/Tank.tscn

```diff
@@ -1,18 +1,14 @@
-[gd_scene load_steps=4 format=2]
+[gd_scene load_steps=2 format=2]
```

Comment:
I kinda rested this node, and made it a "template" to a Tank structure. So every time we want to create a new tank (player or enemy) we just need to `Scene` → `New Inherited Scene...`, check here to see more about that: http://kidscancode.org/blog/2018/01/godot3_inheritance/ :)

(Of course, if you have any questions / something doesn't make sense, ask 😅 )